### ISSUE TITLE
Data source pf bacterial model

### ISSUE THREAD

**Issue by MuMeilin960403** [hzi-bifo/epitope-prediction#12]
Hello, when making predictions, I need to specify the model type, including 'general', 'viral', and 'bacterial'. I saw in the text that EpitopeVec (iBCE-EL) has the best overall performance on broad-spectrum data. Therefore, can we assume that EpitopeVec (iBCE-EL) is the suitable model for the general model? Also, the suitable model for the viral model is undoubtedly svm-viral. However, I haven't seen the training data source for the bacterial model at all. How is svm-bacterial.pickle generated? Could you please help me understand this?

**Comment by akashbahai** [comment]
The viral dataset was created by extracting experimentally verified positive and negative viral epitopes directly from the IEDB database. To ensure accurate evaluation, any peptides in the BCPreds test set that shared more than 80% homology with this new viral training set were removed (please refer to section 3.8 in the paper)
